pocs/utils/images/__init__.py

```diff
@@ -85,30 +89,47 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
 
 
 def _make_pretty_from_fits(fname, **kwargs):
-    config = load_config()
+    header = getheader(fname)
 
-    title = '{} {}'.format(kwargs.get('title', ''), current_time().isot)
+    # get datetime from fits header and make pretty
+    date_time = header.get('DATE-OBS', '')
+    date_time = date_time.replace('T', ' ', 1)
 
-    new_filename = fname.replace('.fits', '.jpg')
+    title = '{} {}'.format(kwargs.get('title', header.get('FIELD', '')),
+                           date_time)
 
+    new_filename = fname.replace('.fits', '.jpg')
     data = getdata(fname)
-    plt.imshow(data, cmap='cubehelix_r', origin='lower')
-    plt.title(title)
-    plt.savefig(new_filename)
 
-    image_dir = config['directories']['images']
+    percent_value = 99.9
 
-    ln_fn = '{}/latest.jpg'.format(image_dir)
+    norm = ImageNormalize(interval=PercentileInterval(percent_value),
+                          stretch=LogStretch())
+    wcs = WCS(fname)
 
-    try:
-        os.remove(ln_fn)
-    except FileNotFoundError:
-        pass
+    if wcs.is_celestial:
+        ax = plt.subplot(projection=wcs)
+        ax.coords.grid(True, color='white', ls='-', alpha=0.3)
 
-    try:
-        os.symlink(new_filename, ln_fn)
-    except Exception as e:
-        warn("Can't link latest image: {}".format(e))
+        ra_axis = ax.coords[0]
+        ra_axis.set_axislabel('Right Ascension')
+        ra_axis.set_major_formatter('hh:mm')
+
+        dec_axis = ax.coords[1]
+        dec_axis.set_axislabel('Declination')
+        dec_axis.set_major_formatter('dd:mm')
+    else:
+        ax = plt.subplot()
+        ax.grid(True, color='white', ls='-', alpha=0.3)
+
+        ax.set_xlabel('X / pixels')
+        ax.set_ylabel('Y / pixels')
+
+    ax.imshow(data, norm=norm, cmap='inferno', origin='lower')
```

Comment:
Also make this slightly more flexible by doing a `cmap=kwargs.get('cmap', 'inferno')`. Then you can play with different colormaps in runtime without touching code.